configure

```diff
@@ -656,16 +656,26 @@ write_action_env_to_bazelrc "TF_CUDNN_VERSION" "$TF_CUDNN_VERSION"
 
 # Configure the compute capabilities that TensorFlow builds for.
 # Since Cuda toolkit is not backward-compatible, this is not guaranteed to work.
+function get_native_cuda_compute_capabilities {
```

Comment:
Sure, this can be helpful, but we need a script that works with linux and windows.
What if we did this on linux, and on windows we simply defaulted to 3.5 and 5.2?
On windows, this script runs under msys